yarn/README.md

```diff
@@ -0,0 +1,6 @@
+# YARN
```

Comment:
In order to get this in, wherever we are able to put it.  We need considerable more documentation.  Look at the examples in the k8s project.  Step by step.